### ISSUE TITLE
After upgrading to RC.11, useRouter () cannot be used in composable

### ISSUE THREAD

**Issue by szulcus** [nuxt/bridge#570]
### Environment

- Operating System: `Windows_NT`
- Node Version:     `v16.17.1`
- Nuxt Version:     `3.0.0-rc.11`
- Nitro Version:    `0.5.4`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `ssr`, `head`, `css`, `plugins`, `components`, `buildModules`, `modules`, `build`, `vite`, `serverHandlers`, `devServerHandlers`, `bridge`
- Runtime Modules:  `-`
- Build Modules:    `()`, `@nuxt/bridge@3.0.0-27740081.058a6c1`

### Reproduction

[nuxt-bridge-rc11.zip](https://github.com/nuxt/bridge/files/9673667/nuxt-bridge-rc11.zip)

### Describe the bug

I used the command `npx nuxi@latest upgrade --force` from https://github.com/nuxt/framework/discussions/7691 and when trying to use `useRouter` I get the following error:
```
[vue-router]: Missing current instance. useRouter() must be called inside <script setup> or setup().
```
`useRouetr` is used only after clicking on the button, when the instance is ready. Everything works fine in RC.8

### Additional context

How can I safely roll back to RC.8?

**Comment by P4sca1** [comment]
The same applies to `useRoute()`. Both are no longer callable in plugins.

**Comment by szulcus** [comment]
> The same applies to `useRoute()`. Both are no longer callable in plugins.

Is this also the case in the official version of Vue 3?

**Comment by szulcus** [comment]
> The same applies to `useRoute()`. Both are no longer callable in plugins.

Is this also the case in the official version of Vue 3?

**Comment by P4sca1** [comment]
I only tested with Vue 2 and Nuxt bridge. I can't tell wether it works with Nuxt / Vue 3.

**Comment by MartinX3** [comment]
Has it something to do with the vue-router 3.6 release adding useRoute() and useRouter() as composables for vue 2.7?

**Comment by danielroe** [comment]
Yes, this is because we switched to using the built-in vue-router composables. (We will need to partially revert that.)

**Comment by BrandonlinU** [comment]
Possible duplicate of #554?
